rollup.config.js

```diff
@@ -0,0 +1,36 @@
+import resolve from '@rollup/plugin-node-resolve'
+import sucrase from '@rollup/plugin-sucrase'
+import mjs from 'rollup-plugin-mjs-entry'
+
+import packageJson from './package.json'
+
+// Matches all dependencies and the nested modules of the dependencies
+const dependencies = Object.keys(packageJson.dependencies).map(
+  dep => new RegExp(`^${dep}/?.*$`)
```

Comment:
The regexp for dependency "foo" would match "foobar". It would need to have the whole `/.*` part be optional, like `(/.*)?`